course/steps/04-events-triggers/en.md

```diff
@@ -1,4 +1,4 @@
-# Events: Using Events as Triggers
+# Using events as triggers
```

Comment:
Changing this title requires changes in the  `config.json`, right? And some steps are in the previous pattern as well, e.g.: Clients: Using Master Data, do you think it makes sense to change all of them?

Reply:
I'll just forget about changing it :hehehe: